x-pack/plugins/security_solution/server/lib/timeline/saved_object/timelines/convert_saved_object_to_savedtimeline.ts

```diff
@@ -17,12 +17,22 @@ import {
   TimelineType,
   TimelineStatus,
 } from '../../../../../common/types/timeline';
+import { DEFAULT_DATA_VIEW_ID, defaultDataViewRef } from '../../../../../common/constants';
 
 // TODO: Added to support legacy TimelineType.draft, can be removed in 7.10
 const TimelineSavedObjectWithDraftRuntime = intersection([
   type({
     id: string,
     version: string,
+    references: array(
```

Comment:
@jonathan-buttner can you please take a look at this? im confused if i need to be doing this, maybe should have been deleted in a merge? idk

Reply:
hmm so does this type represent what will be returned to the UI? I don't think we have been returning the references to the UI. Instead we transform the references into fields within `attributes` or some other field (I think for timeline we strip off `attributes` entirely and just return the contents).